src/server/openapi.yml

```diff
@@ -0,0 +1,446 @@
+          type: string
+        description:
+          type: string
+        keywords:
+          type: array
+          items: {}
+        profileTypes:
+          type: array
+          items: {}
+        isEnabled:
+          type: boolean
+    ApiSearchConfig:
+      type: object
+      properties:
+        BRP:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        AKTES:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        CMS_CONTENT:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        CMS_MAINTENANCE_NOTIFICATIONS:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        KVK:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        KREFIA:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        FOCUS_AANVRAGEN:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        FOCUS_SPECIFICATIES:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        FOCUS_TOZO:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        FOCUS_TONK:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        FOCUS_BBZ:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        FOCUS_STADSPAS:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        WMO:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        VERGUNNINGEN:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        MY_LOCATION:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        AFVAL:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        AFVALPUNTEN:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        BELASTINGEN:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        MILIEUZONE:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        TOERISTISCHE_VERHUUR:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        ERFPACHT:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        SUBSIDIE:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        NOTIFICATIONS:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        CASES:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+    ApiSearchConfigResult:
+      type: object
+      properties:
+        keywords:
+          type: array
+          items: {}
+        keywordsGeneratedFromProps:
+          type: array
+          items: {}
+        description:
+          type: string
+    ServicesAllResponse:
+      type: object
+      properties:
+        AFVAL:
+          type: object
+        AFVALPUNTEN:
+          type: object
+        BRP:
+          type: object
+        BELASTINGEN:

```

Comment:
voor nu kun je dit zo laten maar ik denk dat het goed is als wij deze responses ook definiëren. Andere story.